submissions/the-darkesttt/popup/index.html

```diff
@@ -6,138 +6,154 @@
+                            <span>Apps</span>
+                        </a>
+                    </li>
+                    <li class="popup__inner">
+                        <input class="inner-popup_checkbox" type="checkbox" name="" id="inner-popup_checkbox" tabindex="3">
+                        <label class="inner-popup_checkbox_label" for="inner-popup_checkbox">
+                            <img class="inner-popup_checkbox_button" src="img/arrow-down-sign-to-navigate.svg" alt="">
+                        </label>
+                        <ul class="inner-popup__list">
+                            <li class="popup__item"  tabindex="3">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__icon" src="img/popup-icons/maps.png" alt="">
+                                    <span>Maps</span>
+                                </a>
+                            </li>
+                            <li class="popup__item"  tabindex="3">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__icon" src="img/popup-icons/mic.png" alt="">
+                                    <span>Mic</span>
+                                </a>
+                            </li>
+                            <li class="popup__item"  tabindex="3">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__icon" src="img/popup-icons/news.png" alt="">
+                                    <span>News</span>
+                                </a>
+                            </li>
+                            <li class="popup__item"  tabindex="3">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__icon" src="img/popup-icons/photos.png" alt="">
+                                    <span>Photos</span>
+                                </a>
+                            </li>
+                            <li class="popup__item"  tabindex="3">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__icon" src="img/popup-icons/search.png" alt="">
+                                    <span>Search</span>
+                                </a>
+                            </li>
+                            <li class="popup__item"  tabindex="3">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__icon" src="img/popup-icons/videos.png" alt="">
+                                    <span>Videos</span>
+                                </a>
+                            </li>
+                        </ul>
+                    </li>
                 </ul>
             </nav>
         </header>
     </div>
-
 </body>
-</html>
+</html>
```

Comment:
Add an empty line at the end of the file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)